decentraland/unity-renderer
# feat: biw project list

## What does this PR change?

Fixes #1399 
Fixes #1398 
Fixes #1070

This PR integrates the builder API in the project and list the projects of the user in the builder in world panel

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/biw-project-list&kernel-branch=feat/request-generic-headers
2. Test that builder in world works as normally
3. Test that you can see your builder project in the builder panel

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
